contracts/lbp/LBPManager.sol

```diff
@@ -96,6 +96,7 @@ contract LBPManager {
             _beneficiary != address(0),
             "LBPManager: _beneficiary can not be zero address"
         );
+        require(_tokenList.length == 2, "LBPManager: token list size is not 2");
```

Comment:
rather than having a require check, can we only accept array with 2 elements. that is `IERC20[2] memory _tokenList`

Reply:
The `LBPFactory.create()` call needs a dynamic array. That's why I can't change the function prototype to a fixed size array